src/test/java/gov/usgs/cida/pubs/webservice/pw/PwPublicationMvcServiceHtmlIT.java

```diff
@@ -0,0 +1,111 @@
+
+import java.io.IOException;
+import java.util.List;
+
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultMatcher;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import com.github.springtestdbunit.annotation.DatabaseSetup;
+import com.github.springtestdbunit.annotation.DatabaseSetups;
+
+import gov.usgs.cida.pubs.BaseIT;
+import gov.usgs.cida.pubs.ConfigurationService;
+import gov.usgs.cida.pubs.PubsConstantsHelper;
+import gov.usgs.cida.pubs.busservice.PublicationBusService;
+import gov.usgs.cida.pubs.busservice.pw.PwPublicationBusService;
+import gov.usgs.cida.pubs.dao.PublicationDao;
+import gov.usgs.cida.pubs.dao.intfc.IPwPublicationDao;
+import gov.usgs.cida.pubs.dao.pw.PwPublicationDao;
+import gov.usgs.cida.pubs.domain.LinkType;
+import gov.usgs.cida.pubs.domain.pw.PwPublication;
+import gov.usgs.cida.pubs.domain.pw.PwPublicationLink;
+import gov.usgs.cida.pubs.springinit.DbTestConfig;
+import gov.usgs.cida.pubs.springinit.FreemarkerConfig;
+import gov.usgs.cida.pubs.springinit.SpringConfig;
+import gov.usgs.cida.pubs.springinit.TestSpringConfig;
+import gov.usgs.cida.pubs.utility.CustomStringToArrayConverter;
+import gov.usgs.cida.pubs.utility.CustomStringToStringConverter;
+import gov.usgs.cida.pubs.utility.StringArrayCleansingConverter;
+import gov.usgs.cida.pubs.utility.XmlUtilsTest;
+import gov.usgs.cida.pubs.webservice.MvcService;
+
+@EnableWebMvc
+@AutoConfigureMockMvc(secure=false)
+@SpringBootTest(webEnvironment=WebEnvironment.MOCK,
+	classes={DbTestConfig.class, ConfigurationService.class, TestSpringConfig.class, PwPublicationMvcService.class,
+			PwPublicationBusService.class, LocalValidatorFactoryBean.class,
+			FreemarkerConfig.class, PublicationBusService.class, PwPublication.class,
+			PwPublicationDao.class, PublicationDao.class, SpringConfig.class,
+			CustomStringToArrayConverter.class, StringArrayCleansingConverter.class,
+			CustomStringToStringConverter.class})
+@DatabaseSetups({
+	@DatabaseSetup("classpath:/testCleanup/clearAll.xml"),
+	@DatabaseSetup("classpath:/testData/publicationType.xml"),
+	@DatabaseSetup("classpath:/testData/publicationSubtype.xml"),
+	@DatabaseSetup("classpath:/testData/publicationStream.xml")
+})
+public class PwPublicationMvcServiceHtmlIT extends BaseIT {
```

Comment:
Given that you've mocked the dao, I wouldn't call this an integration test. If you change the name, base class, and remove the database setup it should run faster.